### PR TITLE
Communication frontend <-> backend

### PR DESCRIPTION
**Pull request by BalestraPatrick**

### CODE REVIEW

File: asq-java-q.html [hunk 1]
```diff
@@ -193,44 +188,20 @@
...
-       */
-      clear: function() {
-        if ( ! this.api || ! this.api.clear ) return;
-        this.api.clear();
       },
 
-
-
       /**
        * Collect answers from inner `asq-option`s and orgnize them
```
Comment (triglian) [inline asq-java-q.html:206]: no asq-options anymore :-)

File: test/server/asqJavaqPlugin.test.js [hunk 2]
```diff
@@ -128,90 +130,90 @@ describe("asqMcPlugin.js", function(){
...
+      var result = this.asqJavaq.processEl($, el);
       expect(result._id).to.equal("a-uid");
       expect(result.type).to.equal(this.tagName);
       expect(result.data.stem).to.equal("This is a stem <em>with some HTML</em>");
       expect(result.data.options).to.deep.equal([]);
     });
   });
 
-  describe("parseOptions", function(){
+  describe.skip("parseOptions", function(){
```
Comment (triglian) [inline test/server/asqJavaqPlugin.test.js]: you have to remove the options parsing. I think @marcotollini  is on it

File: test/element/basic.html [hunk 3]
```diff
@@ -3,32 +3,25 @@
...
 
   <script>
     var m = document.querySelector('#m');
-    suite('<asq-multi-choice-q>', function() {
-      test('test default values', function() {
-        assert.equal(m.multi, true);
-      });
+    suite('<asq-java-q>', function() {
 
       test('test default mixin values', function() {
```
Comment (triglian) [inline test/element/basic.html:26]: need. moar. tests!

Test the execute button (that you get what you want and submit it)

Also test the edge cases from when you get a response with the execution results

**Review by triglian — CHANGES_REQUESTED** [review]

File: test/element/basic.html (hunk 3 above)
Comment (BalestraPatrick) [reply test/element/basic.html:26]: Done 👍 
Can we also have a nice Travis green badge now? 😏

Comment (triglian) [reply test/element/basic.html:26]: Yes add something like this https://github.com/PolymerElements/paper-input/blob/master/.travis.yml to the repo, remove the polylint dependency and command and I will add the keys

Comment (BalestraPatrick) [reply test/element/basic.html:26]: Just pushed it 👍
